src/libs/cluster-utils.js

```diff
@@ -82,6 +82,21 @@ export const trimClustersOldestFirst = _.flow(
 
 export const currentCluster = _.flow(trimClustersOldestFirst, _.last)
 
+export const trimAppsOldestFirst = _.flow(
+  _.remove({ status: 'Deleting' }),
+  _.sortBy('createdDate')
+)
+
+export const currentApp = _.flow(trimAppsOldestFirst, _.last)
```

Comment:
I noticed it's possible to get into a situation where you have multiple apps. Should we have some fallback UX for messaging and fixing that? Perhaps adding something to the 'Cloud Environments' page?

Reply:
That sounds good, but sounds like a follow up piece of work probably.